app/lib/three_scale/redis_config.rb

```diff
@@ -3,7 +3,7 @@
 module ThreeScale
   class RedisConfig
     def initialize(redis_config = {})
-      raw_config = (redis_config || {}).symbolize_keys
+      raw_config = (redis_config || {}).deep_symbolize_keys
```

Comment:
We need this because we now accept the new `:ssl_params` config parameter for redis, which is a hash

Reply:
Thanks for the explanation!